common/defines/babylon_defines.txt

```diff
@@ -162,19 +163,19 @@ NDiplomacy = {
 	WAR_GOAL_INFAMY_MAKE_PROTECTORATE = 2.0
 	WAR_GOAL_INFAMY_MAKE_TRIBUTARY = 2.0
 	WAR_GOAL_INFAMY_REDUCE_AUTONOMY = 2.0
-	WAR_GOAL_INFAMY_TRANSFER_SUBJECT = 2.0	
+	WAR_GOAL_INFAMY_TRANSFER_SUBJECT = 2.0
 	WAR_GOAL_INFAMY_ANNEX_COUNTRY = 5.0
 	WAR_GOAL_INFAMY_UNIFICATION = 0.5
 	WAR_GOAL_INFAMY_REGIME_CHANGE = 0.45 #Vanilla .50
 	
 	BASE_YEARLY_INFAMY_DECAY_RATE = 5.0	# Decay rate of infamy per year
 	
-	WAR_GOAL_INFAMY_POPULATION_SCALING_MULTIPLIER = 0.02 #Vanilla 0.03 For each WAR_GOAL_INFAMY_POPULATION_SCALING_FACTOR population in a country/state, multiply infamy by this
-	WAR_GOAL_INFAMY_POPULATION_SCALING_MULTIPLIER_MAX_PER_STATE = 5 # Infamy population multiplier can't be higher than this for each state affected
-	WAR_GOAL_INFAMY_POPULATION_SCALING_MULTIPLIER_MAX_TOTAL = 40 #Vanilla 50 Infamy population multiplier can't be higher than this, regardless of number of states affected
+	WAR_GOAL_INFAMY_POPULATION_SCALING_MULTIPLIER = 0.025 #Vanilla 0.03 For each WAR_GOAL_INFAMY_POPULATION_SCALING_FACTOR population in a country/state, multiply infamy by this
+	WAR_GOAL_INFAMY_POPULATION_SCALING_MULTIPLIER_MAX_PER_STATE = 10 #5 Infamy population multiplier can't be higher than this for each state affected
```

Comment:
Have the new infamy been tested? Doubling the max infamy gained seems.. Excessive.

Reply:
Now that you're able to take major powers and unrecognized major powers as protectorates, that means that potentially a GP could now make a protectorate of Qing or any of the Chinese substates. The Chinese substates gave about 50 infamy, while Qing capped out (for some reason) at 70.5 infamy. Egypt gave about 28 infamy. Japan also capped out at 70.5. So, me and fallonar decided to increase infamy since taking a whole Chinese substate (or the entirety of Qing) should be something that's very punishing.

Reply:
Also this was not the max infamy gained total, that is still a much lower increase. This was max infamy per state, which was way too low for places like China where the pops were so high that it was discounted after around 5 million, less than half in most of those states.

Reply:
Ahh, gotcha! In that case, if the torpedo boats have been tested and they work without errors, I approve :)